{"url": "https://api.github.com/repos/pytorch/pytorch/issues/comments/375986374", "html_url": "https://github.com/pytorch/pytorch/issues/2301#issuecomment-375986374", "issue_url": "https://api.github.com/repos/pytorch/pytorch/issues/2301", "id": 375986374, "node_id": "MDEyOklzc3VlQ29tbWVudDM3NTk4NjM3NA==", "user": {"login": "brando90", "id": 1855278, "node_id": "MDQ6VXNlcjE4NTUyNzg=", "avatar_url": "https://avatars0.githubusercontent.com/u/1855278?v=4", "gravatar_id": "", "url": "https://api.github.com/users/brando90", "html_url": "https://github.com/brando90", "followers_url": "https://api.github.com/users/brando90/followers", "following_url": "https://api.github.com/users/brando90/following{/other_user}", "gists_url": "https://api.github.com/users/brando90/gists{/gist_id}", "starred_url": "https://api.github.com/users/brando90/starred{/owner}{/repo}", "subscriptions_url": "https://api.github.com/users/brando90/subscriptions", "organizations_url": "https://api.github.com/users/brando90/orgs", "repos_url": "https://api.github.com/users/brando90/repos", "events_url": "https://api.github.com/users/brando90/events{/privacy}", "received_events_url": "https://api.github.com/users/brando90/received_events", "type": "User", "site_admin": false}, "created_at": "2018-03-25T17:09:30Z", "updated_at": "2018-03-25T17:09:38Z", "author_association": "NONE", "body_html": "<p><a class=\"user-mention\" data-hovercard-type=\"user\" data-hovercard-url=\"/hovercards?user_id=1310570\" data-octo-click=\"hovercard-link-click\" data-octo-dimensions=\"link_type:self\" href=\"https://github.com/soumith\">@soumith</a> should I open a new git issue or should we address my problem in the forum? I made a question for it already:</p>\n<p><a href=\"https://discuss.pytorch.org/t/how-to-update-pytorch-using-cuda-to-the-most-recent-version/15416/5\" rel=\"nofollow\">https://discuss.pytorch.org/t/how-to-update-pytorch-using-cuda-to-the-most-recent-version/15416/5</a></p>", "body_text": "@soumith should I open a new git issue or should we address my problem in the forum? I made a question for it already:\nhttps://discuss.pytorch.org/t/how-to-update-pytorch-using-cuda-to-the-most-recent-version/15416/5", "body": "@soumith should I open a new git issue or should we address my problem in the forum? I made a question for it already:\r\n\r\nhttps://discuss.pytorch.org/t/how-to-update-pytorch-using-cuda-to-the-most-recent-version/15416/5"}